### ISSUE TITLE
Feedback on tables

### ISSUE THREAD

**Issue by ulde01** [sebgroup/design-library#42]
We have received feedback on tables. These are our thoughts so far: 

- The column header is too discreet, yet gives a "messy" impression. Today it is 12px and Bold. Suggestion: Change column header to 14 px and Medium weight.
- The states (hover, selected) are confusing. Suggestion: Selected row is grey.
- Expanded row is not clear. What just happened? Suggestion: the selected row that is expanded is grey.
- Note that rows are white, not striped white/grey

![image](https://user-images.githubusercontent.com/45568194/54273744-d0a63b00-4586-11e9-948b-c95ff043788f.png)

Description for tables in Design Library: http://designlibrary.sebank.se/components/table/#usage

What more feedback do you have?
/Ulrika, Design management

**Comment by splashdust** [comment]
How should tables look and behavein small viewports?

For public websites we currently use an overflow model, i.e x-axis scroll, with optional fixed right column. We also use a slightly smaller font size in tables on small viewports.

Example:
![table](https://user-images.githubusercontent.com/428824/54282273-f1c55680-459b-11e9-9324-604dda084f3b.gif)
(ignore the grey border that pops up, it is not part of the table)

**Comment by conandx** [comment]
Input from internetsupport when the Account overview is reaching public:
![image](https://user-images.githubusercontent.com/44571006/54341686-bb8ae400-463a-11e9-99c6-b2ee0a1ff372.png)

**Comment by GunillaDE** [comment]
Så här ser det ut på Depåöversikt > Händelser/avräkningsnotor:
![Tabell](https://user-images.githubusercontent.com/48552875/54352242-7920d180-4651-11e9-9607-d5fd0393d247.PNG)

Ett skäl till att det är så trångt, är att kolumnen Händelse ibland behöver mycket utrymme för de texter som kommer från bassystemen. Dessa texter skulle kunna kortas ner, men det är en ganska stor IT-utvecklingsövning.

**Comment by conandx** [comment]
Ja, det är SÅÅÅÅÅ mycket som är helt jäkla tokigt gällande gamla IBP-sidor. De är ommålade i 2-3 generationer, så de håller helt enkelt inte måttet längre. Detta är dock inte ny design, utan bara ommålad design med minimal-effort. 

Om allt vore korrekt kodat så skulle det aldrig hända att kolumner glider ihop så här.... de ska alltid ha padding inne i respektive cell.

**Comment by olgaengvall** [comment]
In Suitability Tool (internal tool) there are tables that look something like this.
![image](https://user-images.githubusercontent.com/43985019/54416530-f276fd80-46ff-11e9-8603-f215b9d6577c.png)

**Comment by didzisspruds** [comment]
The version for mobile viewports I propose to base on a pattern in which the table consists of rows instead of columns. (generic pattern below).

![Mobile table-example](https://user-images.githubusercontent.com/45620670/55221054-57a71480-5211-11e9-9974-cfa3774bde78.jpg)

**Comment by didzisspruds** [comment]
Could you please share some more examples of tables which currently are being used in your projects / systems? Thanks!

**Comment by ulde01** [comment]
In FiRe, they use dropdowns in tables. They are considering using a smaller version of them - height 32 px.

**Comment by Klingis** [comment]
On sebgroup.com we have these tables:
https://sebgroup.com/investor-relations/debt-investors/debt-transactions

![group table 1](https://user-images.githubusercontent.com/49438487/55797124-e8140d80-5acb-11e9-8deb-60139017ce7d.PNG)


And these beautiful ones, imported from an excel file:
https://sebgroup.com/investor-relations/financial-statistics/financial-statistics-group/key-figures

![group table 2](https://user-images.githubusercontent.com/49438487/55797130-ecd8c180-5acb-11e9-899f-5815a63e7114.PNG)

**Comment by Klingis** [comment]
Some inspirational reading btw:
https://uxdesign.cc/design-better-data-tables-4ecc99d23356

**Comment by JessiNygrenWalhed** [comment]
Just a short comment so you know that this is work in progress. The basic feedback about discreet header and so on are quick to fix and make more user friendly 😄 But it is a complex component to explore. Since it needs to be really flexible. @didzisspruds is exploring, and I am trying to... So, if you have any suggestions on how a table should look like, and work. Please keep on giving your input here.

**Comment by didzisspruds** [comment]
Example of funds table in current Baltic's IB. In quite many places currently we use tables with two level headings, so I would say we need a design variation which supports it.

![large-table-LV-example](https://user-images.githubusercontent.com/45620670/58880950-c6a55b80-86e1-11e9-93e1-828cc0032fae.png)

**Comment by ulde01** [comment]
### Component review 2019-10-09
There are more changes needed for tables but this time we only update the absolute most important:
- Bigger table header font: 16 px, medium, Grey 2 (#333333)
- New sorting icon (up, down, neutral): Grey 7 (#B2B3B3) and Dark Blue 2 (#007AC7)

![image](https://user-images.githubusercontent.com/45568194/66406622-373fea00-e9ec-11e9-9bcf-7d7d922fbe23.png)

We will get to the other adjustment at another time. So the rest of what we know about tables stays the same for now.

/Ulrika
Design Management

**Comment by conandx** [comment]
One more table example from a desktop environment.... This one demonstrates the need for a more compact table where it's possible to cram in A LOT of information!
![image](https://user-images.githubusercontent.com/44571006/68772143-961b0380-0629-11ea-85cd-ea89e86e211e.png)
